homeassistant/components/matter/manifest.json

```diff
@@ -6,5 +6,5 @@
   "dependencies": ["websocket_api"],
   "documentation": "https://www.home-assistant.io/integrations/matter",
   "iot_class": "local_push",
-  "requirements": ["python-matter-server==2.1.1"]
+  "requirements": ["python-matter-server==3.0.0"]
```

Comment:
We should bump the add-on before merging here, right?

Reply:
ah, right, sharp as always